documentation/modules/deploying/proc-deploying-kafkaconnector.adoc

```diff
@@ -1,63 +1,96 @@
 // Module included in the following assemblies:
 //
-// assembly-kafka-connect.adoc
+// assembly-deploy-kafka-connect-with-plugins.adoc
 
 [id='proc-deploying-kafkaconnector-{context}']
-= Deploying example KafkaConnector resources
+= Deploying KafkaConnector resources
 
 [role="_abstract"]
-The `KafkaConnector` resource offers a Kubernetes-native approach to management of connectors by the Cluster Operator. 
-Strimzi provides xref:deploy-examples-{context}[example configuration files].
-In this procedure, we use the `examples/connect/source-connector.yaml` file to create the following connector instances as `KafkaConnector` resources:
+Deploy `KafkaConnector` resources to manage connectors.
+The `KafkaConnector` custom resource offers a Kubernetes-native approach to management of connectors by the Cluster Operator.
+You don't need to send HTTP requests to manage connectors, as with the Kafka Connect REST API.
+You manage a running connector instance by updating its corresponding `KafkaConnector` resource, and then applying the updates.
+The Cluster Operator updates the configurations of the running connector instances. 
+You remove a connector by deleting its corresponding `KafkaConnector`.
+
+`KafkaConnector` resources must be deployed to the same namespace as the Kafka Connect cluster they link to.
+
+In the configuration shown in this procedure, the `autoRestart` property is set to `true`.
+This enables automatic restarts of failed connectors and tasks. 
+Up to seven restart attempts are made, after which restarts must be made manually.
+You annotate the `KafkaConnector` resource to xref:proc-manual-restart-connector-str[restart a connector] or xref:proc-manual-restart-connector-task-str[restart a connector task] manually.
+
+NOTE: If you are using a Kafka version earlier than 3.0.0, only a failed connector is restarted and not failed tasks.
+
+.Example connectors
+
+You can use your own connectors or try the examples provided by Strimzi.
+Strimzi provides an xref:deploy-examples-{context}[example `KafkaConnector` configuration file] (`examples/connect/source-connector.yaml`).
```

Comment:
The example file does not work without adding the connectors first. So I wonder if any mention of this should come only after the info from line 34-36.

Reply:
I've rearranged/rewritten this section:

> You can use your own connectors or try the examples provided by Strimzi.
> Up until Apache Kafka 3.1.0, example file connector plugins were included with Apache Kafka. 
> Starting from the 3.1.1 and 3.2.0 releases of Apache Kafka, the examples need to be xref:using-kafka-connect-with-plug-ins-str[added to the plugin path as any other connector]. 
> 
> Strimzi provides an xref:deploy-examples-{context}[example `KafkaConnector` configuration file] (`examples/connect/source-connector.yaml`) for the example file connector plugins, which creates the following connector instances as `KafkaConnector` resources:
> 
> * A `FileStreamSourceConnector` instance that reads each line from the Kafka license file (the source) and writes the data as messages to a single Kafka topic.
> * A `FileStreamSinkConnector` instance that reads messages from the Kafka topic and writes the messages to a temporary file (the sink).
> 
> We use the example file to create connectors in this procedure.